src/test/suite/camel.jbang.version.settings.test.ts

```diff
@@ -19,6 +19,7 @@
 import { ShellExecution, workspace, tasks, Task } from 'vscode';
```

Comment:
3 unused imports to remove ShellExecution, tasks and Task (based on Sonar report)